docs/faststore/docs/reference/project-structure/overview.mdx

```diff
@@ -50,7 +50,7 @@ For more information on configuration options for this file, refer to [Configura
   
 - `tsconfig.json` - Configures the compiler options and project settings for TypeScript, including target version, module resolution, and source file locations.
 
-- `vtex.env` - Holds environment variables that your application can access during runtime, such as API keys and database credentials.
+- `vtex.env` - Stores public environment variables that your application can access during runtime. These include flags like `SITE_HOST` (specifying the store's domain in production) and other publicly accessible values, including feature flags, URLs, and Analytics public tracking IDs. For private environment variables such as API keys and database credentials, use [VTEX secrets](https://v1.faststore.dev/how-to-guides/webops/security/setting-up-secrets).
```

Comment:
suggestion: don't know where exactly, but would be good to mention somewhere that we don't recommend the usage of `.env` to store private keys as it's a security risk, we only recommend using VTEX secrets.